getting-started.md

```diff
@@ -83,6 +83,54 @@ If everything works, you should see your starter app on your device or simulator
   </div>
 </div>
 
+## _Hot Reloading_ and _Full Application Restart_ from within IntelliJ
```

Comment:
And then in conjunction the above, change the heading to `## Viewing source code changes near instantly with 'hot reload'`